staff/staff_views_helper.py

```diff
@@ -540,3 +540,25 @@ def get_my_forms(request, group):
         submitted=False,
         user_id=request.user
     )
+
+
+def create_django_user_from_contact(contact):
+    username = validate_username("%s.%s" % (contact.first_name, contact.last_name))
+    password = DjangoUser.objects.make_random_password()
+    dj = DjangoUser.objects.create_user(
+        first_name=contact.first_name,
+        last_name=contact.last_name,
+        email=contact.email,
+        username=username,
+        password=password,
+        is_staff=False,
+        is_superuser=False,
+        is_active=True
+    )
+    up = UserProfile.objects.get(user=dj)
```

Comment:
Shouldn't `up = dj.userprofile` work here?

Reply:
Yes, implemented in 5bb06ddd59ff0960c7a327460578a3660f172466